fiddler/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
@rajeevthefiddler Please add the `can_connect` service check to this file.

Reply:
Yes, the change is already there in service_checks.json. This is what it looks like. 

[
    {
        "agent_version": "6.0.0",
        "integration":"fiddler",
        "groups": ["host", "instance"],
        "check": "fiddler.can_connect",
        "statuses": ["ok", "critical"],
        "name": "Can Connect",
        "description": "Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored Fiddler instance. Returns `OK` otherwise."
    }
]